src/img/icons/general/tick-rounded.svg

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="13px" height="14px" viewBox="0 0 13 14" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 41 (35326) - http://www.bohemiancoding.com/sketch -->
+    <title>4B5102F7-6782-4D34-8324-F1639EE7FB4E</title>
```

Comment:
With the icons I think we should be stripping out the `title` element in most cases. We do have a Gulp task which optimises the SVG files but it leaves these elements in place as they are sometimes useful/required.